<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>511</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="2">
      <widget class="QSpinBox" name="testsCountSpinBox">
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="testsCountLbl">
       <property name="text">
        <string>How Many Test:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="tstMakerLEdit"/>
     </item>
     <item row="4" column="2">
      <widget class="QSpinBox" name="timeSpinBox">
       <property name="minimum">
        <number>100</number>
       </property>
       <property name="maximum">
        <number>5000</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="timeLbl">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Time(ms):&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="corrCodeLbl">
       <property name="text">
        <string>Correct Code:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="yourCodeLEdit"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="corrCodeLEdit"/>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="corrCodeBtn">
       <property name="text">
        <string>C&amp;hoose</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="statusLbl">
       <property name="text">
        <string>Status:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QTextEdit" name="statusTEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="tstMakerLbl">
       <property name="text">
        <string>Test Maker:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="yourCodeBtn">
       <property name="text">
        <string>&amp;Choose</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QPushButton" name="judgeBtn">
       <property name="text">
        <string>&amp;Judge</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="yourCodeLbl">
       <property name="text">
        <string>Your Code:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="tstMakerBtn">
       <property name="text">
        <string>Ch&amp;oose</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="compileOptionsLbl">
       <property name="text">
        <string>Compile Options</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="compileOptionsLEdit">
       <property name="text">
        <string>-lm</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>511</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="compileOutputBtn">
       <property name="text">
        <string>Com&amp;pile Output</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="lstInputBtn">
       <property name="text">
        <string>Last &amp;Input</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="yourOutBtn">
       <property name="text">
        <string>&amp;Your Output</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="corrOutBtn">
       <property name="text">
        <string>Correct O&amp;utput</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>330</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="resetBtn">
       <property name="text">
        <string>&amp;Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quitBtn">
       <property name="text">
        <string>&amp;Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>525</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>&amp;Menu</string>
    </property>
    <addaction name="actionReset"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionReset">
   <property name="text">
    <string>&amp;Reset</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
